.gitignore

```diff
@@ -6,3 +6,4 @@ dist/*.js
 dist/*.css
 dist/img/*.svg
 data/data.js
+js/lib/id/*.js
```

Comment:
see https://github.com/openstreetmap/iD/commit/90cd412bbe0bf0cfbe0631a8073a5c8d415093f3#commitcomment-15780969 for context